SU-SOE/soe_profile
# D8CORE-2149: Updated tests and sync with stanford_profile.

# READY FOR REVIEW

# Summary
- Removes behat tests and syncs new codeception tests from stanford_profile.
- Original PR and work is here: https://github.com/SU-SWS/stanford_profile/pull/228

# Needed By (Date)
- Next release would be nice

# Urgency
- Low

# Steps to Test

1. Run tests
2. Wait 
4. Wait some more
7. Confirm green lights.

# Affected Projects or Products
- SOE 

# Associated Issues and/or People
- D8CORE-2149

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
